Source/WebCore/platform/mediarecorder/cocoa/MediaRecorderPrivateWriterWebM.mm

```diff
@@ -557,7 +583,7 @@ static String codecStringForMediaVideoCodecId(FourCharCode codec)
 {
     m_firstVideoFrameAudioTime.reset();
     m_resumedVideoTime = MonotonicTime::now();
-    m_resumedAudioTime = m_lastAudioSampleTime;
+    m_resumedAudioTime = m_currentAudioSampleTime;
```

Comment:
Can you describe this change in the commit message?

Reply:
it's part of the other commit, I only included it here for simplicity and not having to rebase all the time